dluhc-component-library/src/components/timetablePage/AccordionDropdown.tsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from "react";
+import { IndicationArrow } from "../inlinesvgs/IndicationArrow";
+
+const AccordionDropdown = ({ information }: { information?: string }) => {
```

Comment:
Can we please pull this type out into its own type `AccordionDropdownProps` like the other files.

Also instead of passing "information" can we use children so that anything can be passed. This just makes the component more reuseable.

It will change the way this component is used to

```
<AccordionDropdown>
    Lorem ipsum...
</AccordionDropdown>
```

or

```
<AccordionDropdown>
    <p className="font-semibold">Lorem ipsum... </p>
</AccordionDropdown>
```

just as an example.